OsmAnd/src/net/osmand/plus/myplaces/tracks/filters/RangeTrackFilter.kt

```diff
@@ -0,0 +1,58 @@
+package net.osmand.plus.myplaces.tracks.filters
+
+import com.google.gson.annotations.Expose
+import net.osmand.plus.OsmandApplication
+import net.osmand.plus.R
+import kotlin.math.max
+import kotlin.math.min
+
+abstract class RangeTrackFilter(
+	val app: OsmandApplication,
+	displayNameId: Int,
+	filterType: FilterType,
+	filterChangedListener: FilterChangedListener)
+	: BaseTrackFilter(displayNameId, filterType, filterChangedListener) {
+	@Expose
+	private var valueFrom = 0f
```

Comment:
Why abstract class has default values from/to?

Reply:
changed